models/response.go

```diff
@@ -16,6 +16,7 @@ type Response struct {
 	Label        string          `json:"label"`
 	Children     []*Element      `json:"children,omitempty"`
 	NoOfChildren int64           `json:"no_of_children,omitempty"`
+	Order        int64           `json:"order,omitempty"`
```

Comment:
In some Java JSON packages the omit empty applied to a field treats `0` as empty so it's not marshalled into the JSON data.

I can't remember the specifics of `omitempty` in the Go package but could we verify the behaviour? I don't know if `0` is a valid value or if its a problem but I think it would be sensible to make sure we fully understand the behaviour and avoid weird hard to identify defects.

Sorry to be a pain - but this type of a bug can be a royal pain to find.

Reply:
good point, I will test this scenario